sklearn/calibration.py

```diff
@@ -274,6 +279,10 @@ def fit(self, X, y, sample_weight=None):
         if sample_weight is not None:
             sample_weight = _check_sample_weight(sample_weight, X)
 
+        for sample_aligned_params in fit_params.values():
+            check_consistent_length(y, sample_aligned_params)
+
+        self.calibrated_classifiers_ = []
```

Comment:
Why is `self.calibrated_classifiers_` defined here?

Reply:
Maybe an error from solving the merge conflict. I will have a look.